src/components/MatchCard/Participants.tsx

```diff
@@ -8,11 +8,13 @@ export const Participants = ({
 }: {
   participants: Participant[]
 }) => {
-  const participant = participants.map((participant) => {
+  const participant = participants.map((participant, index) => {
     const { id } = participant.friend
 
     if (!participant.isWinner) {
-      return <Avatar user={participant.friend} size="sm" key={id} />
+      return (
+        <Avatar user={participant.friend} size="sm" key={`${id}-${index}`} />
```

Comment:
I think we can simply use the participant's id as a key here!

Reply:
I updated to use id and index because we are open to adding the same participant more than once in the match. This was causing the duplicate keys error.